.pre-commit-config.yaml

```diff
@@ -9,8 +9,8 @@ repos:
       - id: end-of-file-fixer
       - id: check-docstring-first
       - id: check-json
+        exclude: 'asv_bench/asv.conf.json'
       - id: check-yaml
-      - id: double-quote-string-fixer
```

Comment:
If we remove this check (which I think would be good), we should enable the default string normalization setting for *Black*:
https://github.com/carbonplan/python-project-template/blob/0dc5bd5b133f931297e694d6f395c1d05f4d0569/pyproject.toml#L4